### PR TITLE
Add Spanish translation for advanced.md

### PR DESCRIPTION
**Pull request by jacostaf10**
Add Spanish translation for `advanced.md` in `fluent`folder

### CODE REVIEW

File: docs/fluent/advanced.es.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+# Avanzado
+
+Fluent se esfuerza por crear una API general y no anclada a ninguna base de datos que te permita trabajar con tus datos. Esto facilita el proceso de aprendizaje de Fluent, sin importar el conector de base de datos que uses. Crear APIs generalizadas también puede hacer que trabajar con tu base de datos en Swift se sienta más cómodo.
```
Comment (alemohamad) [inline docs/fluent/advanced.es.md:3]:
```suggestion
Fluent se esfuerza por crear una API general e independiente de cualquier base de datos que te permita trabajar con tus datos. Esto facilita el proceso de aprendizaje de Fluent, sin importar el conector de base de datos que uses. Crear APIs generalizadas también puede hacer que trabajar con tu base de datos en Swift se sienta más cómodo. 
```

File: docs/fluent/advanced.es.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+|-|-|-|
+|`PostgresDatabase`|[vapor/fluent-postgres-driver](https://github.com/vapor/fluent-postgres-driver)|[vapor/postgres-nio](https://github.com/vapor/postgres-nio)|
+|`MySQLDatabase`|[vapor/fluent-mysql-driver](https://github.com/vapor/fluent-mysql-driver)|[vapor/mysql-nio](https://github.com/vapor/mysql-nio)|
+|`SQLiteDatabase`|[vapor/fluent-sqlite-driver](https://github.com/vapor/fluent-sqlite-driver)|[vapor/sqlite-nio](https://github.com/vapor/sqlite-nio)|
+
+Visita los README de la librerías para más información sobre las APIs específicas de cada base de datos.
+
+### SQL Personalizado
+
+Casi todos los tipos de consultas y esquemas de Fluent soportan un caso `.custom`. Esto te permite utilizar características de bases de datos que Fluent no soporta todavía.
```
Comment (alemohamad) [inline docs/fluent/advanced.es.md:55]:
```suggestion
Casi todos los tipos de consultas y esquemas de Fluent soportan la opción `.custom`. Esto te permite utilizar características de bases de datos que Fluent todavía no soporta.
```

File: docs/fluent/advanced.es.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+} else {
+    // ILIKE no soportado.
+    query.group(.or) { or in
+        or.filter(\.$name == "earth").filter(\.$name == "Earth")
+    }
+}
+query.all()
+```
+
+Las bases de datos SQL soportan tanto `String` como `SQLExpression` en todos los casos `.custom`. El módulo `FluentSQL` ofrece métodos de conveniencia para casos de usos comunes.
```
Comment (alemohamad) [inline docs/fluent/advanced.es.md:73]:
```suggestion
Las bases de datos SQL soportan tanto `String` como `SQLExpression` en todos los casos `.custom`. El módulo `FluentSQL` ofrece métodos convenientes para casos de uso comunes.
```

File: docs/fluent/advanced.es.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+let query = Planet.query(on: req.db)
+if req.db is SQLDatabase {
+    // El conector subyacente de base de datos es SQL.
+    query.filter(.sql(raw: "LOWER(name) = 'earth'"))
+} else {
+    //  La base de datos subyacente _no_ es SQL.
+}
+```
+
+A continuación tienes un ejemplo de `.custom` usando la conveniencia `.sql(raw:)` con el constructor del esquema.
```
Comment (alemohamad) [inline docs/fluent/advanced.es.md:87]:
```suggestion
A continuación tienes un ejemplo de `.custom` usando el método `.sql(raw:)` con el constructor del esquema.
```

**Review by alemohamad — APPROVED** [review]
@jacostaf10 please review this changes, if they also look good to you, merge them and we are ready to go.

**Comment by jacostaf10** [comment]
Thanks for the suggestions @alemohamad! All ready now.

**Pull request merged**